packages/contracts/contracts/optimistic-ethereum/libraries/bridge/Lib_CrossDomainUtils.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+/* Library Imports */
+import { Lib_RLPReader } from "../rlp/Lib_RLPReader.sol";
+
+/**
+ * @title Lib_CrossDomainUtils
+ */
+library Lib_CrossDomainUtils {
+  /**
```

Comment:
nit: comment misaligned here

Reply:
Doesn't the linter catch that sort of issues?